trpcultivate_phenotypes/tests/src/Kernel/Services/Traits/ValidTraitTest.php

```diff
@@ -206,6 +188,34 @@ public function testTraitsServiceDatabaseExpectations() {
     $this->assertEquals($data_type->value, 'Quantitative', 'Unit property - additional type does not match expected value (Quantitative).');
   }
 
+  /**
+   * Test the insertTrait method with schema not null.
+   */
+  public function testInsertTraitWithSchema() {
+    // Set genus to use by the traits service.
+    $this->service_traits->setTraitGenus($this->genus);
+
+    // Get schema name.
+    $schema = $this->chado_connection->getSchemaName();
+
+    // A trait to insert.
+    $trait = 'Trait Name';
+    $method = 'My Method';
+    $unit = 'cm';
+
+    $trait_combo = [
+      'Trait Name' => $trait,
+      'Trait Description' => 'Trait Description',
+      'Method Short Name' => $method,
+      'Collection Method' => 'Some collection method',
+      'Unit' => $unit,
+      'Type' => 'Quantitative',
+    ];
```

Comment:
```suggestion
    // A trait to insert.
    $trait_combo = [
      'Trait Name' => 'My Trait Name',
      'Trait Description' => 'Trait Description',
      'Method Short Name' => 'My Method',
      'Collection Method' => 'Some collection method',
      'Unit' => 'cm',
      'Type' => 'Quantitative',
    ];
```

Variables can be skipped and $trait_combo array can be made simpler.